src/core/basic_utilities/include/arcana/noelle/core/Utils.hpp

```diff
@@ -45,6 +45,16 @@ class Utils {
 
   template <class T>
   static std::vector<T *> sort(const std::set<T *> &s);
+
+  static Value *injectPrint(Value *toPrint,
+                            std::string format,
```

Comment:
For performance reason, you should switch to the following type "const std::string &format".